packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -492,7 +507,7 @@ export default {
       description:
         'Explicitly set simulator to use. Optionally include iOS version between' +
         'parenthesis at the end to match an exact version: "iPhone 6 (10.0)"',
-      default: 'iPhone X',
+      default: '',
```

Comment:
I think we can default to undefined (or just don't provide the default) – can we change that and adjust docs as well?